src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -180,28 +191,31 @@
     private       World                           world             = null;
     //  Updates and Subscriptions
     @NotNull
-    private       Set<EntityPlayerMP>             subscribers       = new HashSet<>();
-    private       boolean                         isDirty           = false;
-    private       boolean                         isCitizensDirty   = false;
-    private       boolean                         isBuildingsDirty  = false;
-    private       boolean                         manualHiring      = false;
-    private       boolean                         isFieldsDirty     = false;
-    private       String                          name              = "ERROR(Wasn't placed by player)";
+    private       Set<EntityPlayerMP>             subscribers      = new HashSet<>();
+    private       boolean                         isDirty          = false;
+    private       boolean                         isCitizensDirty  = false;
+    private       boolean                         isBuildingsDirty = false;
+    private       boolean                         manualHiring     = false;
+    private       boolean                         manualHousing     = false;
+
+    private       boolean                         isFieldsDirty    = false;
+    private       String                          name             = "ERROR(Wasn't placed by player)";
     private BlockPos         center;
     //  Administration/permissions
     @NotNull
     private Permissions      permissions;
     @Nullable
     private BuildingTownHall townHall;
     private int topCitizenId = 0;
-    private int maxCitizens  = Configurations.maxCitizens;
+    private int maxCitizens  = Configurations.gameplay.maxCitizens;
 
     private double overallHappiness = 5;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 5 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)